### PR TITLE
7.0 Meta files

### PR DESCRIPTION
**Pull request by bwrsandman**
Update to #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[![Build Status](https://travis-ci.org/OCA/banking.svg?branch=7.0)](https://travis-ci.org/OCA/banking)
+[![Coverage Status](https://coveralls.io/repos/OCA/banking/badge.png?branch=7.0)](https://coveralls.io/r/OCA/banking?branch=7.0)
+
+Banking addons for Odoo
```
Comment (pedrobaeza) [inline README.md]: In this version, it's still OpenERP

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+reconciliation functionality. While advanced, this functionality will be deprecated in Odoo 8.0 in favour
+of (an extension of) the new, native reconciliation functionality.
+
+    - account_banking_camt - Import of bank statements in the SEPA CAMT.053 format
+
+A number of other modules are available for legacy format bank statement files.
+
+Contact us
+----------
+Join the team on https://launchpad.net/~banking-addons-drivers so that you can subscribe to its mailing list
```
Comment (pedrobaeza) [inline README.md]: This is old data. You can put something like I made on account-invoicing project:

https://github.com/OCA/account-invoicing/blob/7.0/README.md

**Comment by gurneyalex** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**